File: examples/proto/dart/task.dart
import 'monarchic_agent_protocol.pb.dart';

void main() {
  final task = Task()
    ..version = 'v1'
    ..taskId = 'task-123'
    ..role = AgentRole.DEV
    ..goal = 'Implement protocol';

  print(task);
}

File: examples/proto/python/task.py
import monarchic_agent_protocol_pb2 as map_pb2


def main() -> None:
    task = map_pb2.Task(
        version="v1",
        task_id="task-123",
        role=map_pb2.AgentRole.DEV,
        goal="Implement protocol",
    )
    print(task)


if __name__ == "__main__":
    main()
